### ISSUE TITLE
include nix describe-stores in manual

### ISSUE THREAD

**Issue by Ericson2314** [NixOS/nix#5467]
Since that command spits out markdown I thought it already was, but turns out that is not the case.

CC @regnat

**Comment by regnat** [comment]
Yes, it definitely should. But might need a bit more polishing because the output isn’t very readable right now
